blobfuse/utilities.cpp

```diff
@@ -19,6 +19,87 @@ std::string prepend_mnt_path_string(const std::string path)
     return str_options.tmpPath + "/root" + path;
 }
 
+// cleanup function to clean cached files that are too old
+void gc_cache()
+{
+
+    while(true){
+
```

Comment:
i do not see a call to sleep for a case when cleanup deque is not empty but nothing to cleanup. without sleep this thread will consume a whole core.

Reply:
added a sleep in the else statement